cmd/crc/cmd/config/config.go

```diff
@@ -28,6 +29,7 @@ func RegisterSettings(cfg *config.Config) {
 	cfg.AddSetting(Bundle, constants.DefaultBundlePath, config.ValidateBundle, config.SuccessfullyApplied)
 	cfg.AddSetting(CPUs, constants.DefaultCPUs, config.ValidateCPUs, config.RequiresRestartMsg)
 	cfg.AddSetting(Memory, constants.DefaultMemory, config.ValidateMemory, config.RequiresRestartMsg)
+	cfg.AddSetting(DiskSize, 0, config.ValidateDiskSize, config.RequiresRestartMsg)
```

Comment:
Why 0 and not DefaultDiskSize ?

Reply:
Initial concern was to be able to differentiate between "no value was specified on the command line" and "user specified a value which happens to be the default one". Without this, initial iterations of the code would end up calling `if err := setDiskSize(host, cfg.DefaultDiskSize)`, which would trigger an error if the disk had been resized before.
However, now the `setDiskSize` call is wrapped in a `if diskSize != 0` test. If we change this test to `if diskSize != cfg.DefaultDiskSize` then everything should work. I'll test this now.

Reply:
@cfergeau doesn't our validation run initially and error out if the provided disk size is not greater than the default one and we only run the disk resize part when the disk size is provided greater than default?

Reply:
The "only run the disk resize part when the disk size is provided greater than default" is the part which is not done yet, we only have `if startConfig.diskSize != 0`, I think this could be replaced with `if startConfig.diskSize != constants.DefaultDiskSize`, but this needs testing

Reply:
I've force-pushed an updated series, only difference is a `fixup` commit which replaces the `0` default value with a `constants.DefaultDiskSize` default value, we can go with this one I think, I'll squash it.